release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1134,6 +1146,7 @@ link:https://issues.redhat.com/browse/OCPBUGS-39428[(*OCPBUGS-39428*)]
 [id="ocp-4-17-storage-bug-fixes_{context}"]
 ==== Storage
 
+* Previously, the Secrets Store Container Storage Interface (CSI) Driver on HyperShift hosted clusters was failing to mount secrets because of an issue when using the HyperShift CLI to create OpenID Connect (OIDC) infrastructure on {aws-full}. With this release, the issue has been fixed so that the driver can now mount volumes. (link:https://issues.redhat.com/browse/OCPBUGS-18711[*OCPBUGS-18711*])
```

Comment:
```suggestion
* Previously, the Secrets Store Container Storage Interface (CSI) Driver on Hosted Control Plane clusters failed to mount secrets because of an issue when using the CLI to create OpenID Connect (OIDC) infrastructure on {aws-full}. With this release, the issue has been fixed so that the driver can now mount volumes. (link:https://issues.redhat.com/browse/OCPBUGS-18711[*OCPBUGS-18711*])
```
Hypershift is internal; product is Hosted Control Planes. I believe HCPs uses the ROSA CLI? Asked @lahinson

Reply:
:heavy_check_mark: